src/include/MaxxEffect/Control/Direct/MaxxEffect_Revision.h

```diff
@@ -0,0 +1,32 @@
+/*******************************************************************************
+ * Library revision identification
+ ******************************************************************************/
+#ifndef MAXX_EFFECT_REVISION_H
+#define MAXX_EFFECT_REVISION_H
+
+#include "MaxxEffect/MaxxEffect.h"
+#include "MaxxEffect/MaxxStatus.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/*******************************************************************************
+ * Get null-terminated revision string.
+ * 
+ * @param[in]  effect   Initialized effect.
+ * @param[out] revision Revision string.
+ * @param[out] bytes    Revision string size. Optional, use NULL if not needed.
+ * 
+ * @return 0 if success, otherwise fail
+ ******************************************************************************/
+MaxxStatus_t    MaxxEffect_Revision_Get(
+    MaxxEffect_t*       effect,
+    const char**        revision,
+    uint32_t*           bytes);
```

Comment:
Will need to include <stdint.h> for every file that uses the standard int types.

Reply:
done, thanks

Reply:
What if <stdint.h> is included in a header that is included in another file? 
Even then?

Reply:
@stolx Yes, you should include it even there. Do not ever rely on indirect inclusions, because they may change and cause headaches for future work.

Reply:
okay, got it, thanks.